tool/infra-testing-tool/Makefile

```diff
@@ -61,6 +61,11 @@ update_ami:
 		-var 'server_go_version=$(SERVER_GO_VERSION)' \
 		.
 
+generate_proto:
```

Comment:
This rule should depend on the proto files and be a dependency for the build rule I think.

Reply:
totally, but I will use static path for now, since we only have one proto file, I will make something smarter if we get more protos